test/spec/modules/guAnalyticsAdapter_spec.js

```diff
@@ -271,7 +271,9 @@ describe('Gu analytics adapter', () => {
     events.emit(CONSTANTS.EVENTS.NO_BID, NOBID_EVENT);
     const ev = analyticsAdapter.context.queue.peekAll();
     expect(ev).to.have.length(4);
-    expect(ev[3]).to.be.eql({
+    const nobidEv = ev[3];
+    expect(nobidEv).to.have.property('ttr');
+    expect(nobidEv).to.include({
       ev: 'nobid',
       n: 'trustx',
       bid: '41cba721ab11b',
```

Comment:
Shouldn't this include the ttr property as well?

Reply:
Wouldnt it make the test unstable?

Reply:
Sorry, I misunderstood what `include` means!